symfony/mercure-bundle/0.1/manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+    "bundles": {
+        "Symfony\\Bundle\\MercureBundle\\MercureBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "container": {
+        "env(MERCURE_PUBLISH_URL)": "",
```

Comment:
Why add them as defaults in the container when you have them set up in env? 🤔

Reply:
To prevent an error when env var are not defined (when building a Docker container for instance).

Reply:
You can instead define the `env(MERCURE_PUBLISH_URL)` parameter in the bundle.

Reply:
It’s not the bundle’s responsibility: you may use it without env cars.
Also, the use may choose to use other names of env vars.